internal/commands/stack/suggest.go

```diff
@@ -0,0 +1,82 @@
+package stack
+
+import (
+	"bytes"
+	"html/template"
+	"sort"
+
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/logging"
+)
+
+type suggestedStack struct {
+	ID          string
+	Description string
+	Maintainer  string
+	BuildImage  string
+	RunImage    string
+}
+
+var suggestedStacks = []suggestedStack{
+	{
+		ID:          "heroku-18",
+		Description: "The official Heroku stack based on Ubuntu 18.04",
+		Maintainer:  "Heroku",
+		BuildImage:  "heroku/pack:18-build",
+		RunImage:    "heroku/pack:18",
+	},
+	{
+		ID:          "io.buildpacks.stacks.bionic",
+		Description: "A minimal Paketo stack based on Ubuntu 18.04",
+		Maintainer:  "Paketo Project",
+		BuildImage:  "paketobuildpacks/build:base-cnb",
+		RunImage:    "paketobuildpacks/run:base-cnb",
+	},
+	{
+		ID:          "io.buildpacks.stacks.bionic",
+		Description: "A large Paketo stack based on Ubuntu 18.04",
+		Maintainer:  "Paketo Project",
+		BuildImage:  "paketobuildpacks/build:full-cnb",
+		RunImage:    "paketobuildpacks/run:full-cnb",
+	},
+	{
+		ID:          "io.paketo.stacks.tiny",
+		Description: "A tiny Paketo stack based on Ubuntu 18.04, similar to distroless",
+		Maintainer:  "Paketo Project",
+		BuildImage:  "paketobuildpacks/build:tiny-cnb",
+		RunImage:    "paketobuildpacks/run:tiny-cnb",
+	},
+}
+
+func newSuggestCmd(logger logging.Logger) *cobra.Command {
```

Comment:
```suggestion
func suggest(logger logging.Logger) *cobra.Command {
```
Our command pattern is typically to do the command name itself (e.g. `command.Build`), so I think this would fit the pattern a bit more.